api/src/main/java/org/apache/cloudstack/api/command/user/backup/DeleteBackupCmd.java

```diff
@@ -61,6 +62,13 @@ public class DeleteBackupCmd  extends BaseAsyncCmd {
             description = "id of the VM backup")
     private Long backupId;
 
+    @Parameter(name = ApiConstants.FORCED,
+            type = CommandType.BOOLEAN,
+            required = false,
+            description = "force the deletion of backup",
```

Comment:
Can you please change the description to something like "force the deletion of backup which removes the entire backup chain but keep VM in Backup Offering"? Otherwise, the force parameter looks to me like "force delete just this backup in any case".

Reply:
Done, thanks.